src/tests/ftest/launch.py

```diff
@@ -747,8 +747,11 @@ def get_test_files(test_list, args, yaml_dir):
 
     """
     # Replace any placeholders in the extra yaml file, if provided
+    combined_extra_yaml = None
     if args.extra_yaml:
-        args.extra_yaml = replace_yaml_file(args.extra_yaml, args, yaml_dir)
+        for extra_yaml in args.extra_yaml:
+            combined_extra_yaml = replace_yaml_file(extra_yaml, args, yaml_dir)
+        args.extra_yaml = combined_extra_yaml
```

Comment:
args.extra_yaml should retain its original list typing:
```
args.extra_yaml = [replace_yaml_file(extra, args, yaml_dir) for extra in args.extra_yaml]
```

And later on replace lines 764-765 with just:
```
    command.extend(args.extra_yaml)
```

Reply:
Oh, avocado accepts multiple :)

Reply:
This way means each `args.extra_yaml` will essentially contain duplicate base yaml. And then later on, we'll be passing all those:
https://github.com/daos-stack/daos/blob/0b7796cff2eae0cc3a6b9ed0366b1436194a707a/src/tests/ftest/launch.py#L1305-L1306
Functionally, it seems okay, but overkill.
Note `args.extra_yaml` is referenced in both `get_test_files` and `run_tests`.